### ISSUE TITLE
Bug: 最新版本build报错 无法正常打包

### ISSUE THREAD

**Issue by imdap** [vbenjs/vue-vben-admin#5194]
### Version

Vben Admin V5

### Describe the bug?

最新版本build报错
![image](https://github.com/user-attachments/assets/d8e2366d-71ec-4946-8974-aadd28744570)

### Reproduction

pnpm reinstall
pnpm build:play

### System Info

```shell
Binaries:
    Node: 20.15.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 10.7.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 9.12.2 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.22621.3527
```

### Validations

- [X] Read the [docs](https://doc.vben.pro/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by imdap** [comment]
运行也一样报错 无法启动项目
![image](https://github.com/user-attachments/assets/ca1f5899-f3e6-4bc1-a759-5a5084173609)

**Comment by imdap** [comment]
临时解决办法
https://github.com/evanw/esbuild/issues/4010#issuecomment-2556446069
测试可正常打包

出现问题原因是vite引用的esbuild自动升级到0.24.1造成的问题

**Comment by imdap** [comment]
vite里引用的esbuild自动升级导致的  0.24.1会导致出现该问题
使用https://github.com/evanw/esbuild/issues/4010#issuecomment-2556446069方案可解决

**Comment by likui628** [comment]
fixed 24a4935e85a410f81e9e4afcfb607cbc8662e36b
